docs/.link-check-errors.txt

```diff
@@ -2,5 +2,6 @@ Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
 - https://mui.com/size-snapshot
+- https://mui.com/x/react-data-grid/editing/#row-editing
```

Comment:
This will require an update of the material-ui repo. Especially the `PricingTable.tsx`

Reply:
Yes, after releasing the v6 stable